### DISCUSSION TITLE
Hidden toolbar buttons

### DISCUSSION THREAD

**Discussion by chrkaatz** [outline/rich-markdown-editor#305] [General] [answered]
Is there still a possibility to disable some buttons in the toolbar?

There was a commit to hide some toolbar buttons (#35) and another to add documentation (https://github.com/outline/rich-markdown-editor/issues/92).

**Comment by tommoor** [comment, marked as answer]
The functionality was removed as part of the `11.0` release. If you want to customize then I'd recommend a more flexible package like [remirror](https://github.com/remirror/remirror)
